CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## What's New in 3.2.0
+
+### Improvements 
+
+- PAT authentication is now supported for JIRA. 
```

Comment:
Might be a silly question - but does it actually work for JIRA cloud? Or is it exclusively a server thing?

Reply:
Found the answer - it's [only DC and Server](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html). Perhaps we could clarify here?
![image](https://github.com/user-attachments/assets/df220c0d-2e4a-4ca6-bad4-f5dd9f94b417)

Reply:
I think it should work for the cloud. We can give it a test. 

Cloud supports PAT.